OpenRA.Mods.Common/Traits/Render/WithIdleOverlay.cs

```diff
@@ -20,11 +20,14 @@ namespace OpenRA.Mods.Common.Traits.Render
 	[Desc("Renders a decorative animation on units and buildings.")]
 	public class WithIdleOverlayInfo : PausableConditionalTraitInfo, IRenderActorPreviewSpritesInfo, Requires<RenderSpritesInfo>, Requires<BodyOrientationInfo>
 	{
-		[SequenceReference]
+		[Desc("Image used for this decoration. Defaults to the actor's type.")]
+		public readonly string Image = null;
```

Comment:
The linter reports a lot of Warnings (see https://travis-ci.org/github/OpenRA/OpenRA/builds/679823665#L1418). Might this be the cause?

Reply:
Yes, I am not sure what the warning is about. I would vote for removal as leaving this null is a valid entry.